core/backend/src/CheckpointManager.ts

```diff
@@ -225,7 +225,7 @@ export class V2CheckpointManager {
           // eslint-disable-next-line @typescript-eslint/no-floating-promises
           logPrefetch(CloudSqlite.startCloudPrefetch(container, dbName, { minRequests, nRequests: maxRequests, timeout }));
         } else {
-          Logger.logInfo(loggerCategory, `Skipping prefetch due to size limits.`, { maxBlocks, totalBlocksInDb: dbStats?.totalBlocks, v2props });
+          Logger.logInfo(loggerCategory, `Skipping prefetch due to size limits or ongoing prefetch.`, { maxBlocks, numPrefetches: dbStats?.nPrefetch, totalBlocksInDb: dbStats?.totalBlocks, v2props });
```

Comment:
probably helpful to say whether we're skipping due to an active prefetch already in progress or it being disabled.

Reply:
Yeah I thought about having separate log messages for each case, but as it is currently we can deduce which case we are in by looking at what was logged. The maxBlocks limit, totalBlocks in the database, and the number of active prefetches. Between those 3, we can figure out the case. Do you still prefer a separate log message?

Reply:
as long as you have something you can search for in the logs to find when multiple prefetches would have happened, that's fine